IBM/carbon-addons-iot-react
# fix(composedmodal): removed double scrollbars

Closes #1352

**Summary**
Removed the double scrollbars that could appear on the ComposedModal when the error message was displayed.

![image](https://user-images.githubusercontent.com/5167091/85878672-5f20e900-b7d9-11ea-9369-138af6459bfe.png)


**Change List (commits, features, bugs, etc)**
- Conditionally added a reduced bottom-margin to the modal body
- updated a story so that it could be verified
- wrote unit test

**Acceptance Test (how to verify the PR)**

1. Go to /story/watson-iot-composedmodal--error-states
2. add a massive text to the body content knob, e.g."_Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum._"
3. Verify that there will be no double scrollbars when you change the height of the modal by resizing the browser or addons panel.
4. Verify that the other stories works as before
